src/containers/BooksList.css

```diff
@@ -1,3 +1,84 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 38px;
+  padding: 15px 20px;
+  background-color: #fff;
+}
+
+.header {
+  margin-left: -10px;
+  display: flex;
+  list-style: none;
+  align-items: center;
+  height: 45px;
+}
+
+.header__link {
+  font-family: sans-serif, Montserrat;
+  cursor: pointer;
+  font-weight: normal;
+  letter-spacing: 1.9px;
+  font-size: 13px;
+  height: 16px;
+  margin: 20px;
+  opacity: 0.5;
+}
+
+.header__logo {
+  margin: 0 auto;
+  height: 30px;
+  font-size: 26px;
+  letter-spacing: normal;
+  color: #0290ff;
+  font-weight: bold;
+  opacity: 1;
+}
+
+.header__book_link {
+  opacity: 1;
+}
+
+.user {
+  width: 25px;
+  height: 25px;
+  margin-right: 26px;
+  padding: 5px;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-self: center;
+}
+
+.user-logo {
+  background-image: url('../images/user.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  width: 23px;
+  height: 23px;
+}
+
+@media (max-width: 750px) {
+  .user {
+    display: none;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: column;
+  }
+
```

Comment:
It would be nice to work on this before merging.
As mentioned in the previous review, we need to write our CSS in best practice, that is;  not repeating a rule.

1. we have a few rules using `display: flex`
2. repeated `justify-content: center`
3. repeated `align-item`

we could have it like:
```
.header-container, .header,  {
   display: flex
}

```

Reply:
Thanks